packages/app/src/settings/SettingsContainer.cy.tsx

```diff
@@ -50,4 +50,11 @@ describe('<SettingsContainer />', { viewportHeight: 800, viewportWidth: 900 }, (
 
     cy.findByText(defaultMessages.settingsPage.projectId.title).should('not.exist')
   })
+
+  it('renders footer with CTA button', () => {
```

Comment:
I used a component test instead of an e2e test to get some experience with CT 😄 in general, when both approaches are equal, do we have a preference for one or the other?

Reply:
For verifying text content I prefer to keep it in the component tests.